packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -255,22 +255,21 @@ function fbw.read_configs()
 end
 
 -- Apply configuration for a new network ( used in ubus daemon)
-function fbw.apply_user_configs(configs, hostname)
-    fbw.log('apply_file_config(ssid=' .. configs.ssid .. ', hostname=' .. hostname .. ')')
+function fbw.create_network(ssid, hostname, password)
+    fbw.log('apply_file_config(ssid=' .. ssid .. ', hostname=' .. hostname .. ')')
     local uci_cursor = config.get_uci_cursor()
-    -- Mesh network name
-    local name = configs.ssid
-    -- Format hostname
-    hostname = hostname or config.get("system", "hostname")
+
     -- Save changes in lime-community
-    uci_cursor:set("lime-community", 'wifi', 'ap_ssid', name)
-    uci_cursor:set("lime-community", 'wifi', 'apname_ssid', name..'/%H')
-    uci_cursor:set("lime-community", 'wifi', 'adhoc_ssid', 'LiMe.%H')
-    uci_cursor:set("lime-community", 'wifi', 'ieee80211s_mesh_id', 'LiMe')
```

Comment:
what happened with these two? shouldn't be used still?

Reply:
I don't understan why they where needed to be here in the first place. Maybe there were no defaults?
Currently the defaults (in `/lime-system/files/etc/config/lime-defaults`) are:
`option ieee80211s_mesh_id 'LiMe'`
`option adhoc_ssid 'LiMe'`
Note that the adhoc_essis default wa different in FBW. I also don't know really why. Anyway, adhoc is not used by FBW so it is useless here.

Reply:
agreed.

Reply:
related issue: https://github.com/libremesh/lime-packages/issues/612